nav2_smac_planner/README.md

```diff
@@ -115,6 +115,7 @@ planner_server:
       cost_travel_multiplier: 2.0         # For 2D: Cost multiplier to apply to search to steer away from high cost areas. Larger values will place in the center of aisles more exactly (if non-`FREE` cost potential field exists) but take slightly longer to compute. To optimize for speed, a value of 1.0 is reasonable. A reasonable tradeoff value is 2.0. A value of 0.0 effective disables steering away from obstacles and acts like a naive binary search A*.
       angle_quantization_bins: 64         # For Hybrid nodes: Number of angle bins for search, must be 1 for 2D node (no angle search)
       analytic_expansion_ratio: 3.5       # For Hybrid/Lattice nodes: The ratio to attempt analytic expansions during search for final approach.
+      analytic_expansion_max_length: 3.0    # For Hybrid/Lattice nodes: The maximum length of the analytic expansion to be considered valid to prevent unsafe shortcutting
```

Comment:
I would also mention that it is in meters, and that it should be scaled together with minimum_turning_radius.

Reply:
Added

Reply:
It's also more detailed in the configuration guide on the website in the docs PR that will be merged at the same time https://github.com/ros-planning/navigation.ros.org/pull/234/files#diff-4b80e4229af9d6c9cb965894da700179259b3dc1925f28641a3cc024c29adeeeR91